Lib/test/test_symtable.py

```diff
@@ -54,33 +181,70 @@ class SymtableTest(unittest.TestCase):
     top = symtable.symtable(TEST_CODE, "?", "exec")
     # These correspond to scopes in TEST_CODE
     Mine = find_block(top, "Mine")
+
     a_method = find_block(Mine, "a_method")
     spam = find_block(top, "spam")
     internal = find_block(spam, "internal")
     other_internal = find_block(spam, "other_internal")
     foo = find_block(top, "foo")
-
+    Alias = find_block(top, "Alias")
+    GenericAlias = find_block(top, "GenericAlias")
+    # TODO(RUSTPYTHON)
```

Comment:
```suggestion
    # XXX: RUSTPYTHON
```
We don't have a strict rule for this, but conventionally using this to distinguish local patch from failing tests.